analyses/infercnv-consensus-cell-type/references/diagnosis-celltype-groups.tsv

```diff
@@ -0,0 +1,15 @@
+diagnosis_group	celltype_groups
+B-cell leukemia	dendritic cell, innate lymphoid cell, macrophage, monocyte, myeloid, natural killer cell, plasma cell, T cell, endothelial cell, pericyte
+T-cell leukemia	B cell, dendritic cell, innate lymphoid cell, macrophage, monocyte, myeloid, natural killer cell, plasma cell, endothelial cell, pericyte
+Myeloid leukemia	B cell, dendritic cell, innate lymphoid cell, macrophage, natural killer cell, plasma cell, T cell
+Mixed-type leukemia	dendritic cell, innate lymphoid cell, macrophage, natural killer cell, plasma cell, endothelial cell, pericyte
+Sarcoma	B cell, dendritic cell, innate lymphoid cell, macrophage, monocyte, myeloid, natural killer cell, plasma cell, T cell, endothelial cell, pericyte
+Brain & CNS	B cell, dendritic cell, innate lymphoid cell, macrophage, monocyte, myeloid, natural killer cell, plasma cell, T cell, endothelial cell, pericyte, fibroblast
+Adrenocortical carcinoma	B cell, dendritic cell, innate lymphoid cell, macrophage, monocyte, myeloid, natural killer cell, plasma cell, T cell, endothelial cell, pericyte, fibroblast

```

Comment:
I would remove fibroblasts from any carcinomas or solid tumors that have cells that might resemble fibroblasts. That means at minimum we should remove them from ACC. I'm on the fence on if they should be included in NB, but I think I've convinced myself it's okay to keep there.